logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/conversation/GetAllContactsNotInConversationUseCase.kt

```diff
@@ -5,14 +5,16 @@ import com.wire.kalium.logic.data.id.QualifiedID
 import com.wire.kalium.logic.data.user.OtherUser
 import com.wire.kalium.logic.data.user.UserRepository
 import com.wire.kalium.logic.functional.fold
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.map
 
 class GetAllContactsNotInConversationUseCase internal constructor(
     private val userRepository: UserRepository
 ) {
-    suspend operator fun invoke(conversationId: QualifiedID) =
+    suspend operator fun invoke(conversationId: QualifiedID): Flow<Result> =
```

Comment:
```suggestion
    suspend operator fun invoke(conversationId: ConversationId): Flow<Result> =
```

Also, we take the opportunity to add some `kdocs` about this use case purpose :D  
What do you think ?